js/libs/core.js

```diff
@@ -189,6 +190,14 @@ Fliplet.FormBuilder = (function() {
         return this.showLabel || this.showLabel === undefined;
       };
 
+      component.computed.getLocalFormattedValue = function() {
+        if (!this.value) {
```

Comment:
why is this block required? the next return is the same as this one

Reply:
@squallstar your right thanks, fixed